frontend/src/slices/auth/auth.slice.ts

```diff
@@ -1,4 +1,5 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { type UserStore } from 'shared/build/packages/users/users.js';
```

Comment:
Don't use direct imports from shared, user types should be re-exported though `frontend/src/packages/users/libs/types/types.ts`